lib/plugins/threaded-signing/plugin.c

```diff
@@ -193,10 +200,8 @@ signing_worker_thread(void *user_data)
           // SV_MEMORY.
```

Comment:
The comment states that you set status to SV_MEMORY, but that is not true. Rewrite.